packages/vertica-nodejs/test/integration/client/api-tests.js

```diff
@@ -9,7 +9,7 @@ suite.test('null and undefined are both inserted as NULL', function (done) {
   pool.connect(
     assert.calls(function (err, client, release) {
       assert(!err)
-      client.query('CREATE LOCAL TEMP TABLE my_nulls(a varchar(1), b varchar(1), c integer, d integer, e date, f date)')
+      client.query('DROP TABLE IF EXISTS my_nulls; CREATE TABLE my_nulls(a varchar(1), b varchar(1), c integer, d integer, e date, f date)')
```

Comment:
Why can't this be a local temp table?

Reply:
I had set it this way for examining in vsql, but just now tested this with local temp and it failed to read the row for some reason. Some issue with temp tables and null values? Not sure. I think I'd like to merge this fix and if there's another issue there log a lower-priority ticket for it.

Reply:
okay, that works for me.